lib/models/postAction.js

```diff
@@ -58,6 +62,10 @@ function doIt(action, event, callback) {
         } else if (options.text) {
             requestOptions.body = options.text;
         }
+        else {
+        	//Added default else clause
+            logger.debug('doIt() - Default else clause');	
```

Comment:
Review indent.

Reply:
Fixed in d978818056a0293666063e39270979bdb5adb8ab